### DISCUSSION TITLE
Actions, how disable censored logs?

### DISCUSSION THREAD

**Discussion by sambP** [community/community#40447] [Actions] [answered]
### Select Topic Area

Question

### Body

Our actions currently suffer from this output. We need to know the exit code. Is it possible to disable dotted outputs?

```
error Command failed.
Exit code: ***
```

**Comment by airtower-luna** [comment, marked as answer]
If the exit code is automatically censored that implies you have a secret with a value that's identical to the exit code (or possibly very similar, I don't know if they use any heuristic there). Can you remove that secret?

**Reply by sambP** [reply]
thx!

**Comment by telmop** [comment]
I'm not logging secrets yet I'm seeing censored logs. It's censoring parts of strings. Is there a way to disable this for debugging? I tried setting the variables in https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging and it didn't work
